chain/indexer/src/streamer/mod.rs

```diff
@@ -63,7 +73,6 @@ fn test_problematic_blocks_hash() {
 /// This function supposed to return the entire `StreamerMessage`.
 /// It fetches the block and all related parts (chunks, outcomes, state changes etc.)
 /// and returns everything together in one struct
-#[async_recursion]
```

Comment:
Please, don't ask me why it was here I don't remember and can come up with the answer